app/assets/v2/js/hackathon-projects.js

```diff
@@ -29,8 +29,12 @@ const submitProject = (logo, data, callback) => {
     if (callback) {
       callback(response);
     }
+    if (document.location.pathname.includes('fulfill')) {
+      document.location.reload();
+    }
```

Comment:
Wouldn't this lead to recursion with no breakout?

Reply:
no because this post action is just to create a project not to submit the submission form once the project exist this post will not be call anymore.